app.json

```diff
@@ -47,8 +47,8 @@
       "required": false,
       "value": "secret"
     },
-    "V1SCHEMA": {
-      "description": "DB schema selected whe no version (or version 1) requested",
+    "SCHEMA": {
+      "description": "DB schema selected for routes",
       "required": false,
       "value": "1"
```

Comment:
I believe this is for your sample production app, this might break it? Also, maybe consider changing the value from `"1"` to something else, depends on how easy that would be with your setup.